### DISCUSSION TITLE
冰风谷在deepin 15.5启动错误

### DISCUSSION THREAD

**Discussion by linusboyle** [Gamuxorg/bbs#342] [游戏或网站运行反馈]
站上似乎有很多游戏是用安卓模拟器搞得？我的deepin 15.5 运行报错，看上去是模拟器的问题
> libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  156 (GLX)
  Minor opcode of failed request:  3 (X_GLXCreateContext)
  Value in failed request:  0x0
  Serial number of failed request:  89
  Current serial number in output stream:  90

**Comment by shouhuanxiaoji** [comment]
不是，全部是原生的游戏。
这个似乎是显卡驱动问题，你是什么显卡，装的什么驱动

**Comment by linusboyle** [comment]
940MX,NVIDIA的开源驱动

**Comment by shouhuanxiaoji** [comment]
装一下闭源驱动试试

**Comment by linusboyle** [comment]
我今天自己鼓捣好了，是AppRun出了问题
加上libjson.so.0 ，自己写个脚本解决了。
